recipes/qt/6.x.x/conandata.yml

```
@@ -86,6 +86,12 @@ sources:
       - "https://ftp.yz.yamagata-u.ac.jp/pub/qtproject/archive/qt/6.5/6.5.3/single/qt-everywhere-src-6.5.3.tar.xz"
     sha256: "7cda4d119aad27a3887329cfc285f2aba5da85601212bcb0aea27bd6b7b544cb"
 patches:
+  "6.8.0":
+    - "base_path": "qtwebengine"
+      "patch_description": "Workaround for too long .rps file name"
+      "patch_file": "patches/c72097e_6.8.0.diff"
```

```suggestion
      "patch_file": "patches/c72097e_6.6.0.diff"
```
I think you can keep using the existing patch?